src/olympia/amo/tests/__init__.py

```diff
@@ -766,6 +794,8 @@ def version_factory(file_kw=None, **kw):
     max_app_version = kw.pop('max_app_version', '5.0.99')
     version = kw.pop('version', '%.1f' % random.uniform(0, 2))
     application = kw.pop('application', amo.FIREFOX.id)
+    if not kw.get('license') and not kw.get('license_id'):
+        kw['license'] = license_factory(**(kw.get('license_kw', {})))
```

Comment:
Should we create a default re-usable license instead, to avoid stuffing the test db with useless licenses when creating multiple add-ons ?